test/commons/math/get-offset.js

```diff
@@ -1,104 +1,66 @@
+  it('returns center to corner or square when at a diagonal', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 30px; height: 30px; margin: 0; position: absolute; top: 50px; left: 50px">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[1];
+    const nodeB = fixture.children[3];
+    assert.closeTo(getOffset(nodeA, nodeB), 61.5, round);
   });
 
-  it('returns the distance to the edge when elements overlap on an edge', function () {
-    var fixture = fixtureSetup(
-      '<a style="padding-right:30px; display:inline-block">' +
-        '<b style="width:30px; height: 20px; display:inline-block">&nbsp;</b>' +
-        '</a>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = nodeA.children[0];
-    assert.closeTo(getOffset(nodeA, nodeB), 30, round);
-    assert.closeTo(getOffset(nodeB, nodeA), 30, round);
+  it('returns 0 if nodeA is overlapped by nodeB', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 30px; height: 30px; margin: 0; position: absolute; top: -10px; left: -10px">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[1];
+    const nodeB = fixture.children[3];
+    assert.equal(getOffset(nodeA, nodeB), 0);
   });
 
-  it('returns the shortest side of the element when an element overlaps on a corner', function () {
-    var fixture = fixtureSetup(
-      '<a style="padding:30px 30px 0 0; display:inline-block">' +
-        '<b style="width:30px; height: 20px; display:inline-block">&nbsp;</b>' +
-        '</a>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = nodeA.children[0];
-    assert.closeTo(getOffset(nodeA, nodeB), getDiagonal(30), round);
-    assert.closeTo(getOffset(nodeB, nodeA), 20, round);
+  it('returns 0 if nodeB is overlapped by nodeA', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 30px; height: 30px; margin: 0; position: absolute; top: -10px; left: -10px">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[3];
+    const nodeB = fixture.children[1];
+    assert.equal(getOffset(nodeA, nodeB), 0);
   });
 
-  it('returns smallest diagonal if elmA fully covers elmB', function () {
-    var fixture = fixtureSetup(
-      '<a style="padding:10px; display:inline-block">' +
-        '<b style="width:20px; height: 30px; display:inline-block">&nbsp;</b>' +
-        '</a>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = nodeA.children[0];
-    assert.closeTo(getOffset(nodeA, nodeB), getDiagonal(10), round);
-    assert.closeTo(getOffset(nodeB, nodeA), 10, round);
+  it('allows passing radius', () => {
```

Comment:
This test name isn't very clear. Maybe:

```suggestion
  it('subtracts minNeighbourRadius from center-to-center calculations', () => {
```